create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConstants.java

```diff
@@ -0,0 +1,13 @@
+package no.sikt.nva.scopus;
+
+import nva.commons.core.JacocoGenerated;
+
+public class ScopusConstants {
+    //ids and fields:
+    public static final String ADDITIONAL_IDENTIFIERS_SCOPUS_ID_SOURCE_NAME = "ScopusId";
```

Comment:
In NVA we use "ID" only for URIs, and "Identifier" for unique strings or other types of identifiers

Reply:
The field 'id' in NVA is reserved for uses where it is aliased for JSON-LD's '@id'

Reply:
Fikset

Reply:
I think that "ScopusId" will be key in a json map and therefore it cannot start with capital. I suggest the name "scopusIdentifier"